checkstyle-sonar-plugin/src/main/resources/org/sonar/plugins/checkstyle/rules.xml

```diff
@@ -1245,6 +1245,10 @@
     <name><![CDATA[Package Declaration]]></name>
     <configKey><![CDATA[Checker/TreeWalker/PackageDeclaration]]></configKey>
     <status>READY</status>
+    <cardinality>MULTIPLE</cardinality>
```

Comment:
I missed your point, and did a release.
lets keep it as is , I will be more accurate/attentive later on.